src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -711,7 +729,7 @@ export class PageControllerBase {
     }
   }
 
-  feedbackUrlFromRequest(request: Request): string | void {
+  feedbackUrlFromRequest(request: Request): string | undefined {
```

Comment:
Cheers for this

Can we remove the duplicate method from **SummaryPageController** now?

It also extends from **PageControllerBase** but has its own version:

https://github.com/DEFRA/forms-runner/blob/86e70c53d5c2d93cbd59760ea0759b467f4cc651/src/server/plugins/engine/pageControllers/SummaryPageController.ts#L200

Reply:
Aha, you have read my mind 😄 

I have already done this in a follow-up PR: https://github.com/DEFRA/forms-runner/pull/228

I went down a rabbit hole and it ended up getting too big so I spun it out separately.